scripts/upgrade-prequisities/devtron-upgrade-init.yaml

```diff
@@ -0,0 +1,96 @@
+kind: Job
+metadata:
+  name: devtron-upgrade-init
+  namespace: devtroncd
+spec:
+  backoffLimit: 0
+  activeDeadlineSeconds: 7200
+  template:
+    spec:
+      tolerations:
+        - operator: "Exists"
+      containers:
+      - name: upgrade-initializer
+        image: mirror.gcr.io/irawal007/kubectl:28f6c5c6-8489-66897
+        env:
+          - name: PGHOST
+            value: postgresql-postgresql.devtroncd
+          - name: PGPORT
+            value: "5432"
+          - name: PGUSER
+            value: postgres 
+          - name: PGPASSWORD 
+            valueFrom:
+              secretKeyRef:
+                name: postgresql-postgresql
+                key: postgresql-password
+          - name: BACKUP_DIR
+            value: /backup-data 
+          - name: BACKUP_FILENAME
+            value: backup.sql 
+          - name: DB_TO_CHECK
+            value: clairv4
+        command: ["/bin/bash", "-c"]
+        args:
+          - |
+            BACKUP_TARGET="${BACKUP_DIR}/${BACKUP_FILENAME}"
+            echo "[$(date)]   Host: ${PGHOST}:${PGPORT}"
+            echo "[$(date)]   User: ${PGUSER}"
+            sleep 5
+            echo "Terminating all the connections to database and shutting down Devtron temporarily"
+            kubectl scale deploy -n devtroncd --all --replicas=0
+            kubectl scale deploy -n argo --all --replicas=0
+            kubectl get sts -n devtroncd -o name | grep -v "statefulset.apps/postgresql-postgresql" | xargs -I{} kubectl scale {} -n devtroncd --replicas=0
+            while true; do
+                running_devtroncd=$(kubectl get pods -n devtroncd | grep -vE "postgresql-postgresql|app-sync-cronjob-|app-manual-sync-job-|devtron-upgrade-init-|app-sync-job-" | grep -c "Running")
+                running_argo=$(kubectl get pods -n argo | grep -c "Running")
+                total_running=$((running_devtroncd + running_argo))
+                if [[ "$total_running" -eq 0 ]]; then
+                    echo "[$(date)] ✅ INFO: All pods are scaled down."
+                    break
+                fi
+                echo "[$(date)] 🔄 INFO: Still waiting... ($total_running pods running)"
+                sleep 5
+            done
+            echo "Checking the size of existing Database"
+            psql -h "$PGHOST" -U "$PGUSER" -c "SELECT datname, pg_size_pretty(pg_database_size(datname)) FROM pg_database;" >> db_11.txt
+            cat db_11.txt
+            DBS=("casbin" "git_sensor" "lens" "clairv4" "orchestrator")
+            for db in "${DBS[@]}"; do
+              echo "Terminating connections to $db database..."
+              PGPASSWORD="$PGPASSWORD" psql -h "$PGHOST" -U "$PGUSER" -c "SELECT pg_terminate_backend(pg_stat_activity.pid) FROM pg_stat_activity WHERE datname='$db';"
+            done
+            mkdir -p ${BACKUP_DIR}
+            if [ "$DB_EXISTS" = "1" ]; then
+              echo "[$(date)] Database '${DB_TO_CHECK}' found. Performing full dump (pg_dumpall)..."
```

Comment:
Change `Performing full dump (pg_dumpall)...` to `Performing full dump (pg_dumpall) excluding '${DB_TO_CHECK}' ...`